src/renderer/redux/actions/settings.js

```diff
@@ -4,6 +4,9 @@ import Fs from 'fs';
 import Http from 'http';
 
 import Lbry from 'lbry';
+import moment from 'moment';
+
+const UPDATE_IS_NIGHT_INTERVAL = 60 * 1000;
```

Comment:
I'd prefer to check every 10 minutes. No need to be so accurate.

so `60 * 10 * 1000`